app/Http/Controllers/UserListController.php

```diff
@@ -70,14 +70,14 @@ public function delete($listid){
         return \Redirect::back();
     }
 
-    public function add_game(Request $request, $listid, $gameid){
-        if(\Auth::check()){
+    public function add_game(Request $request, $listid, $gameid) {
```

Comment:
__[code_sniffer]__
* Method name "UserListController::add_game" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->